xapian-core/tests/Makefile.am

```diff
@@ -82,8 +82,8 @@ check-honey: apitest$(EXEEXT)
 endif
 
 ## Test programs to be run
-TESTS = apitest$(EXEEXT) internaltest$(EXEEXT) stemtest$(EXEEXT) \
- unittest$(EXEEXT)
+TESTS = run_honey run_none run_inmemory run_glass \
+	internaltest$(EXTEXT) stemtest$(EXTEXT) unittest$(EXTEXT)
```

Comment:
If we do end up merging this, I notice this changes `$(EXEEXT)` to `$(EXTEXT)`, which is wrong - it's **EXEEXT** meaning the **exe**cutable file **ext**ension - on Unix it's empty (so your typo is harmless there) but on platforms where executables have a `.exe` extension this will break things.